rflx/model/session.py

```diff
@@ -1,25 +1,16 @@
 from typing import Dict, List, Sequence
 
+from rflx import declaration as decl, expression as expr, statement as stmt
```

Comment:
Are there conflicts that speak against importing the required classes directly?

Reply:
No, it is mainly about avoiding complexity. Long lists of imports makes it hard to refactor code. Manually adding and removing imports after refactoring a module is cumbersome and time-consuming (I also made this experience when refactoring `generator` and `parser`). That's why I try to switch to importing modules instead of single classes/functions, especially for modules with a high number of classes (like the ones here).